src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file1 = new File(fromFileName);
+        File file2 = new File(toFileName);
+        int supply = 0;
+        int buy = 0;
+
+        //1st part: reading from file
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file1));) {
```

Comment:
)) I didn't meant to do it like this, create three separate methods that will be doing operations that I described in the previous review